QuoteOfTheDayQuickStart/README.md

```diff
@@ -0,0 +1,374 @@
+# QuoteOfTheDayQuickStart
+
+### Prerequisites
+
+This quick start assumes you have completed a previous step in order to create the following required resources:
+
+* An instance of Azure App Configuration
+* An instance of Azure Application Insights
+* A variant feature flag named "Greeting" inside the App Configuration instance
+  * One variant named 'On' with value 'true'
+  * One variant named 'Off' with value 'false'
+  * Allocation between 'On' and 'Off' set to 50% / 50%
+
+### Steps
+
+1.
+
+In cmd run
```

Comment:
As discussed, can we add descriptions of what the step is doing? Can we also put the description and bullet number on the same line?

(For all the steps down below)